src/components/stepforms/ToDateStepForm.vue

```diff
@@ -6,15 +6,31 @@
       v-model="editedStep.column"
       name="Column to convert:"
       :options="columnNames"
-      placeholder="Add columns"
-      data-path=".columns"
+      placeholder="Add column"
+      data-path=".column"
       :errors="errors"
     />
-    <InputTextWidget
-      v-if="this.translator === 'mongo40'"
-      class="dateFormat"
-      v-model.number="editedStep.format"
+    <AutocompleteWidget
+      v-if="translator === 'mongo40'"
```

Comment:
I don't understand why you want to hide this if it's not mongo ?

Reply:
Because mongo3.6 does not support for providing a format. It only tries to infer.
See https://docs.mongodb.com/manual/reference/operator/aggregation/dateFromString/
(the `format` parameter is only supported from version 4.0)

I will add a comment though (there is one in the steps technical doc but it's good to have it in the UI component too)